docs/docsite/rst/dev_guide/developing_inventory.rst

```diff
@@ -329,7 +329,7 @@ A skeleton example of this JSON object is::
 				"ungrouped"
 			]
 		},
-		"ungrouped": {}
+		"ungrouped": []
```

Comment:
a list is still incorrect, it should be a dictionary, but with a "children" key to be recognized as a group

Reply:
Ah, well as long as the documentation is adapted as it should be, I'm fine :)